### DISCUSSION TITLE
Is there the chance to change the radio button color?

### DISCUSSION THREAD

**Discussion by LauGroupie95** [buildo/bento-design-system#297] [Q&A] [answered]
Hi @gabro and @veej,
is there the chance to change the radio button color? As a default it is transparent.

What I want to achieve:
![image](https://user-images.githubusercontent.com/63547770/180439481-e608fe7c-2520-4a01-8d85-90c0cf7187a4.png)

What I have:
![image](https://user-images.githubusercontent.com/63547770/180439563-0d42e229-1b83-45f4-bed9-5bccde72a03c.png)

Playroom:
https://playroom.bento.buildo.io/#?code=N4Igxg9gJgpiBcIA8AhCAPABAIwIZgGsBzAJwgFcA7KAXgB0QBnCAMwBcA1ASwgBsY2DAHx1KmTEgBKuKDwDiZcgAcAYlxi8oo8eMq4AtjBrAGDAL7adAN1y9yRkyACCAOQAi5y%2BIiUAwgAtcSiIHAAoASkwaIUxgMwsxHR8UOxJjCKiYuISdTF5cbA1jBgB5JRgSXDYIEk9E7yU2HkpGYwBtLx1gPIKNeEwGVw8QABpMGzsYfsH3BkwzEc7xbvzC3mmQEskGMYn7Da25hc6AXRykknVKNirm%2BhB-Gq4ALx8b3lNEgHoRSiQvtDoX6jEAAdy4UDY-kYCDaAEYACwIgAMZyAA

Thank you in advance

**Comment by gabro** [comment]
Interesting, it may actually be a bug. @Phyele what do you think? Should it be white by default or is it correct that it is transparent?

**Reply by Phyele** [reply]
Correct, all selection controls should have Background/Primary fill.

**Comment by Phyele** [comment, marked as answer]
We added it as a bug [here](https://github.com/buildo/bento-design-system/issues/298)
